STT/train.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+set -xe
+
+pushd ${STT_DIR}
+	all_train_csv="$(find /mnt/extracted/data/ -type f -name '*train.csv' -printf '%p ' | sed -e 's/ $//g')"
+	all_dev_csv="$(find /mnt/extracted/data/ -type f -name '*dev.csv' -printf '%p ' | sed -e 's/ $//g')"
+	all_test_csv="$(find /mnt/extracted/data/ -type f -name '*test.csv' -printf '%p ' | sed -e 's/ $//g')"
+
+	mkdir -p /mnt/sources/feature_cache || true
+
+	# Do not overwrite checkpoint file if model already exist: we will likely
+	# only package
+	if [ -f "/transfer-checkpoint/checkpoint" -a ! -f "/mnt/models/output_graph.tflite" ]; then
+		echo "Using checkpoint from ${TRANSFER_CHECKPOINT}"
+		# use --load_checkpoint_dir for transfer learning
+		LOAD_CHECKPOINT_FROM="--load_checkpoint_dir /transfer-checkpoint --save_checkpoint_dir /mnt/checkpoints"
+	else
+		LOAD_CHECKPOINT_FROM="--checkpoint_dir /mnt/checkpoints/"
+	fi;
+
+	EARLY_STOP_FLAG="--early_stop true"
+	if [ "${EARLY_STOP}" = "0" ]; then
+		EARLY_STOP_FLAG="--early_stop false"
+	fi;
+
+	AMP_FLAG=""
+	if [ "${AMP}" = "1" ]; then
+		AMP_FLAG="--automatic_mixed_precision true"
+	fi;
+
+	SKIP_BATCH_TEST_FLAG=""
+	if [ "${SKIP_BATCH_TEST}" = "1" ]; then
+		SKIP_BATCH_TEST_FLAG="--skip_batch_test true"
+	fi;
+
+	# Basic augmentation for data
+	# TODO: Add use of overlays with noise datasets
+	# ^ This would require to download and prepare noise data
+	ALL_AUGMENT_FLAGS=""
+	if [ "${ENABLE_AUGMENTS}" = "1" ]; then
+		ALL_AUGMENT_FLAGS='--cache_for_epochs 10'
```

Comment:
nit: maybe passing this as a parameter to docker (source from a file)? might be a good idea

Reply:
Like so:

```zsh
echo "--cache_for_epochs 10" >> augments.txt
echo "--augment reverb[p=0.1,delay=50.0~30.0,decay=10.0:2.0~1.0]" >> augments.txt
# All augments in augments.txt...

docker run  [ --args ] `< augments.txt`
```

I need to test this before anthing.

Reply:
Or we could stick to POSIX guidelines and use an argument `--augmentation_file` (or something like that) where you pass a file containing arguments to use and copy it in the image and let `train.sh`, `test.sh`, `evaluate.sh` run a new small script to load the arguments and return them for `$ALL_AUGMENT_FLAGS`.

Reply:
It looks like this now:

`augemnts.txt`
```
--cache_for_epochs 10
--augment reverb[p=0.1,delay=50.0~30.0,decay=10.0:2.0~1.0]
resample[p=0.1,rate=12000:8000~4000]
codec[p=0.1,bitrate=48000:16000]
volume[p=0.1,dbfs=-10:-40]
pitch[p=0.1,pitch=1~0.2]
tempo[p=0.1,factor=1~0.5]
frequency_mask[p=0.1,n=1:3,size=1:5]
time_mask[p=0.1,domain=signal,n=3:10~2,size=50:100~40]
dropout[p=0.1,rate=0.05]
add[p=0.1,domain=signal,stddev=0~0.5]
multiply[p=0.1,domain=features,stddev=0~0.5]
```

`train.sh`:
```bash
	# Basic augmentation for data
	# TODO: Add use of overlays with noise datasets
	# ^ This would require to download and prepare noise data
	ALL_AUGMENT_FLAGS=""
	if [ "${ENABLE_AUGMENTS}" = "1" ]; then
		parse_augments_args.sh
	fi;
```

We could make a PR to bring this into eval and test scripts.

Reply:
We should probably also update the docs